README.md

```diff
@@ -42,10 +42,22 @@ make k8s=1.29
 # build an AMI with a specific Kubernetes version and a specific OS distro
 make k8s=1.29 os_distro=al2023
 
+# build Neuron AMI with a specific Kubernetes version
+make k8s=1.29 os_distro=al2023 accelerator_vendor=neuron
+
+# build Nvidia AMI with a specific Kubernetes version and default driver major version
+make k8s=1.29 os_distro=al2023 accelerator_vendor=nvidia
+
+# build Nvidia AMI with a specific Kubernetes version and a specific driver major version
+make k8s=1.29 os_distro=al2023 accelerator_vendor=nvidia nvidia_major_driver_version=550
```

Comment:
Is 550 provided in the nvidia repo? I think we should add something around for how to check available versions

Reply:
My bad, 555 and 560 are available. Checking for available versions might be tricky, we need to check what is available in the repo.